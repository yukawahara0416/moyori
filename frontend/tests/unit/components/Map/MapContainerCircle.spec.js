import { shallowMount } from '@vue/test-utils'
import Component from '@/components/Map/MapContainerCircle.vue'

let wrapper
let propsData

beforeEach(() => {
  propsData = {
    center: { lat: 36.204824, lng: 138.252923 }
  }

  wrapper = shallowMount(Component, {
    propsData,
    stubs: ['gmap-circle']
  })
})

afterEach(() => {
  wrapper.destroy()
})

describe('props', () => {
  it('center', () => {
    expect(wrapper.props().center).toStrictEqual(propsData.center)
    expect(wrapper.props().center instanceof Object).toBe(true)
  })
})

describe('template', () => {
  it('snapshot', () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})
