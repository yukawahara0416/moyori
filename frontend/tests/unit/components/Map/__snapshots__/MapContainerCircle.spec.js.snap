// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template snapshot 1`] = `
<gmap-circle-stub
  center="[object Object]"
  options="[object Object]"
  radius="500"
/>
`;
